### DISCUSSION TITLE
Next 10 - Funding Deep Dive

### DISCUSSION THREAD

**Discussion by marco-ippolito** [nodejs/help#4625] [Announcements]
Hello 👋🏼 The Next-10 team has planned a new deep dive on funding for contributors.

We will discuss topics like:

 - How can the project accept and handle contributions of $
 - Use of crowdfunding 
   
Related issue: https://github.com/nodejs/next-10/issues/273 

## Date and Time

June 13th 2024 
Time: 11-1 ET (17-18 CEST)

## Location 
https://zoom.us/j/99950131676

cc @nodejs/collaborators @nodejs/next-10

**Comment by Trott** [comment]
@rginn @bensternthal

**Comment by Trott** [comment]
This seems like a solution, and without the articulation of a problem it's trying to solve, it seems like a solution in search of a problem.

Do we need more money? If so, I think we should ask the Foundation for it.

Do we have problems that crowdfunding would solve? I'm unaware of any.

There's "We have a backlog and we need to incentivize people to address it." That's https://github.com/nodejs/TSC/issues/1553, but there seems to be significant opposition.

**Comment by wesleytodd** [comment]
> I think we should ask the Foundation for it

I thought this was clarified in the past but maybe I am wrong. The foundation does not just have funds for projects beyond some specific programs like the travel fund, grants, etc. In earlier conversations I remember specifically discussing the role of the foundation *not* being funding specific work streams and instead being a support role, so I am not sure this ask is ever intended to be the way things operate.

> This seems like a solution, and without the articulation of a problem it's trying to solve, it seems like a solution in search of a problem.
> Do we have problems that crowdfunding would solve? I'm unaware of any.

The deep dive is intended to dig into this question. So to say it is a solution at this phase is maybe just a miss-understanding of the goal of doing a deep dive discussion? According to the thread linked, there is mixed opinion so to me that clearly justifies holding a discussion like this issue proposes. It is hard to tell, but are you opposing having this deep dive or more just expressing your opposition to seeking funding at all (which is a valid opinion, and one I would hope you would bring to the discussion).

**Reply by Trott** [reply]
> It is hard to tell, but are you opposing having this deep dive or more just expressing your opposition to seeking funding at all (which is a valid opinion, and one I would hope you would bring to the discussion).

I was hoping to get more context for the conversation, but it's OK if it's not there yet.  If they're useful, perhaps what I wrote can be used for a "what we hope to accomplish with this conversation" goal for the meeting.

**Reply by wesleytodd** [reply]
Oh sorry I should have replied as a thread, I always forget to do that with discussions.

Ok cool, thanks for the clarification. I have missed the last few meetings, but afaik the goal is just to host a discussion so folks can form more clear opinions on what the goals might be, what could use direct funding, what could we do to promote individual contributors seeking funding, etc. So sounds like we are on the same page now and will try to make sure the conversation doesn't skip the step of having a problem to solve. 😄

**Comment by marco-ippolito** [comment]
@nodejs/collaborators @nodejs/next-10 we are trying to revamp the deep dive please add your availability
https://doodle.com/meeting/participate/id/dyJErLVe

**Reply by wesleytodd** [reply]
Is this still open? I was on a long vacation and am just catching up on GH notifications now but wanted to attend.

**Reply by Qard** [reply]
If I recall correctly, @mhdawson had said not enough people signed up and so it was cancelled. I too would still like to see this happen though.

**Reply by marco-ippolito** [reply]
Not enough partecipants, I tried to organize this twice without reaching a more than 5 rsvp, Im no longer interested in organizing it a third time

**Reply by sheplu** [reply]
do we want to try to have a go at it for the next collaborator summit? I guess we would have enough people in situ + some joining remotely?

**Reply by marco-ippolito** [reply]
yes I think that would probably be better

**Reply by wesleytodd** [reply]
Yeah totally understandable to be frustrated when folks dont show up. I have been struggling to keep up with node things with all the express stuff going on, and then I was OOO for a month this summer over the time this was scheduled. Maybe now is just not the right time?

**Reply by mhdawson** [reply]
Scheduling for the collaborator summit seems like a great idea to me.
